### ISSUE TITLE
Can't get the API key to install the package

### ISSUE THREAD

**Issue by Storm-Develop** [tangrams/tangram-unity#93]
![screenshot_15](https://user-images.githubusercontent.com/11896104/41065773-58306050-69a5-11e8-8400-9b1fe98c271a.png)
The link to sign up or sign in is down.

**Comment by matteblair** [comment]
Yeah, sorry about that. Mapzen is no longer providing developer services as of January 31, 2018: https://mapzen.com/blog/shutdown/ This means the vector tile server is offline as well.

You can check out https://www.nextzen.org/ instead for a replacement service. 

I'll leave this issue open so I remember to update the README and tile URLs.

**Comment by Storm-Develop** [comment]
Sorry to hear that=( Thank you

**Comment by Krakers114** [comment]
I refactored it, so now it works with Nextzen. Here is the PR link: https://github.com/tangrams/tangram-unity/pull/94
